shared/js/background/experiments.es6.js

```diff
@@ -0,0 +1,43 @@
+const settings = require('./settings.es6')
+const retentionExperiments = require('../../data/experiments-out')
+
+class Experiment {
+    constructor () {
+        this.variant = ''
+        this.activeExperiment = {}
+    }
+
+    getVariant () {
+        const atbVal = settings.getSetting('atb')
+        if (atbVal && atbVal[atbVal.length - 2].match(/[a-z]/i)) {
```

Comment:
Can you make sure it's a valid atb. There's a pattern in atb.js that you can copy over. https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/js/background/atb.es6.js#L12

Reply:
Good call, added